### PR TITLE
JMX - OpenTelemetry Docs

### PR DESCRIPTION
**Pull request by himanshu219**
## Purpose of this pull request

This pull request (PR) creates new page for JMX - OpenTelemetry app

Issue number: 

[SUMO-218988](https://sumologic.atlassian.net/browse/SUMO-218988)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/jmx.png')} alt="Thumbnail icon" width="50"/>
+
+Java Management Extensions (JMX) is a standard component of the Java Platform. JMX gives developers a standard and simple way to manage resources, including services, devices, and applications. JMX is dynamic, so you can manage and monitor resources as soon as they are created, implemented, or installed.
+
+The Sumo Logic App for **JMX** allows you to analyze and gain insights about Java applications. The dashboards provide a quick glance at various deployment metrics like memory, CPU, GC performance, and thread behavior, so you can troubleshoot unexpected behavior in your Java environment and the applications running in it.
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:16]:
```suggestion
The Sumo Logic app for JMX allows you to analyze and gain insights about Java applications. The dashboards provide a quick glance at various deployment metrics like memory, CPU, GC performance, and thread behavior, so you can troubleshoot unexpected behavior in your Java environment and the applications running in it.
```

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+
+## Metric Types
+
+The Sumo Logic App for JMX collects metrics from Java applications, via the [JMX Receiver for opentelemetry.](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver)
+
+The following types of metrics are collected from JMX:
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:23]:
```suggestion
## Metric types

The Sumo Logic App for JMX collects metrics from Java applications via the [JMX Receiver for OpenTelemetry](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver).

The following types of metrics are collected from JMX:
```

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+Below is the input required:
+
+- **JMX Endpoint**: The URL of the broker endpoint (default: `localhost:9092`).
+- **Jar File Path**: Enter the path to the OpenTelemetry JMX Metric Gatherer file configured in the prerequisites section.
+- **UserName**: Username for JMX authentication, if applicable.
+- **Password**: Password for JMX authentication, if applicable.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<!-- <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Kafka-OpenTelemetry/Kafka-YAML.png' alt="YAML" /> -->
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:109]: do you have the JMX version of this image?

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+  Below are docs for popular java applications:
+
+    * [Active MQ](https://activemq.apache.org/jmx)
+    * [Kafka](https://kafka.apache.org/documentation/#remote_jmx)
+    * [Apache Tomcat](https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html#Enabling_JMX_Remote)
+    * [Cassandra](https://cassandra.apache.org/doc/4.1/cassandra/operating/security.html#jmx-access)
+    * [HBase](https://hbase.apache.org/metrics.html)
+    * [Solr](https://solr.apache.org/guide/8_3/using-jmx-with-solr.html#configuring-jmx)
+    * [Zookeeper](https://zookeeper.apache.org/doc/r3.4.2/zookeeperJMX.html#ch_starting)
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:82]:
```suggestion
If you want to set up authentication, you'll need to set the below system variables:

```
-Dcom.sun.management.jmxremote.authenticate=true
-Dcom.sun.management.jmxremote.password.file=<application conf directory>/jmx.password
-Dcom.sun.management.jmxremote.access.file=<application conf directory>/jmx.access
```

 Below are docs for popular Java applications:

* [Active MQ](https://activemq.apache.org/jmx)
* [Kafka](https://kafka.apache.org/documentation/#remote_jmx)
* [Apache Tomcat](https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html#Enabling_JMX_Remote)
* [Cassandra](https://cassandra.apache.org/doc/4.1/cassandra/operating/security.html#jmx-access)
* [HBase](https://hbase.apache.org/metrics.html)
* [Solr](https://solr.apache.org/guide/8_3/using-jmx-with-solr.html#configuring-jmx)
* [Zookeeper](https://zookeeper.apache.org/doc/r3.4.2/zookeeperJMX.html#ch_starting)
```

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+## Collecting Metrics and Installing App for JMX
+
+The process to set up collection for JMX Metrics is done through the following steps.
+
+### Step 1: Set up OpenTelemetry Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Kafka-OpenTelemetry/Kafka-Collector.png' alt="Collector" />
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:94]: Should this say JMX, not Kafka?

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+1.  Copy the yaml to the `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  Restart the collector using:
+  ```sh
+  Restart-Service -Name OtelcolSumo
+  ```
+
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml to the `/etc/otelcol-sumo/conf.d/` folder in the Kafka instance which needs to be monitored.
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:144]: https://github.com/SumoLogic/sumologic-documentation/pull/2734/files#r1254659426

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+<img src={useBaseUrl('img/integrations/app-development/jmx.png')} alt="Thumbnail icon" width="50"/>
+
+Java Management Extensions (JMX) is a standard component of the Java Platform. JMX gives developers a standard and simple way to manage resources, including services, devices, and applications. JMX is dynamic, so you can manage and monitor resources as soon as they are created, implemented, or installed.
+
+The Sumo Logic App for **JMX** allows you to analyze and gain insights about Java applications. The dashboards provide a quick glance at various deployment metrics like memory, CPU, GC performance, and thread behavior, so you can troubleshoot unexpected behavior in your Java environment and the applications running in it.
+
+
+## Metric Types
+
+The Sumo Logic App for JMX collects metrics from Java applications, via the [JMX Receiver for opentelemetry.](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver)
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:21]:
```suggestion
## Metrics types

The Sumo Logic App for JMX collects metrics from Java applications via the [JMX Receiver for OpenTelemetry](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver).
```

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+## Fields creation in Sumo Logic for MySQL
+
+  sumo.datasource - Has fixed value of jmx
+  jmx.endpoint - The endpoint of the activemq broker
+
+## Prerequisites
+
+The JMX Receiver uses the OpenTelemetry JMX Metric Gatherer. For more details, [see their docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver#details).
+
+* Download the jar from [releases](https://github.com/open-telemetry/opentelemetry-java-contrib/releases) and place it in `/opt/` or in `C:\ProgramData`. Remember the path of the jar it will be used in configuring JMX app.
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:49]:
```suggestion
* Download the jar from [releases](https://github.com/open-telemetry/opentelemetry-java-contrib/releases) and place it in `/opt/` or in `C:\ProgramData`. Remember the path of the jar, which you'll use to configure the JMX app.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md (hunk 3 above)
Comment (himanshu219) [reply docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:98]: I will update the image for this

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md (hunk 5 above)
Comment (himanshu219) [reply docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:83]: the image is same so I didn't changed it to reuse it. should I made a copy for it?

Comment (kimsauce) [reply docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:82]: Oh good call! nm, it's ok to stay as-is.

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+
+### Step 1: Set up OpenTelemetry Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Kafka-OpenTelemetry/Kafka-Collector.png' alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for JMX Collection.
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:86]:
```suggestion
In this step, we'll configure the yaml required for JMX Collection.
```

File: docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+* Understand the number of objects collected across memory pools and its impact on the Heap memory.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/JMX-OpenTelemetry/JMX-Garbage-Collector.png' alt="Garbage Collector" />
+
+### Class Loading and Threads
+
+The **JMX - Class Loading and Threads** dashboard shows key information about the number and type of threads deadlocked, peak, and GC threads of your java virtual machine running on the deployment.
+
+Use this dashboard to:
+* Identify abnormal spikes in Threads and Loaded Classes
```
Comment (kimsauce) [inline docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md:188]:
```suggestion
* Identify abnormal spikes in Threads and Loaded Classes.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**